### ISSUE TITLE
Check accessibility for the site

### ISSUE THREAD

**Issue by tonypesdm** [tonypesdm/CAMS#35]
Feedback for client: Form labels for the various fields across the system haven't been created correctly. eg all fields on the login/registration page. Labels exist, but haven't been associated with the corresponding field, producing two accessibility errors - "Orphaned form label" and "Missing form label".

**Comment by joerichardsESDM** [comment]
We have a number of label + input filed combinations where the 'for' property on the label tag does not match the ID property on the input tag. This is because the input tags are rendered out from ASP Textbox controls and get an auto-generated client ID applied. We can work around this, but will need care on pages with the potential for duplicate IDs.
